templates/mysqlbackup.sh.erb

```diff
@@ -27,7 +31,9 @@ PATH=<%= @execpath %>
 
 
 
+<%- if @kernel == 'Linux' -%>
 set -o pipefail
```

Comment:
If simplification would be the *main* goal, I'd always opt to change the script to rely on `/bin/sh` instead of bash for obvious compatibility reasons. Otherwise I'm fine with my suggested fallback on non-Linux platforms...